docs/remote/devcontainer-cli.md

```diff
@@ -11,6 +11,26 @@ DateApproved: 12/17/2021
 
 Given the growing number of use cases for dev containers, there is a companion `devcontainer` command line interface (CLI) that can be used independent of the [Remote - Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) or GitHub Codespaces. This article will walk you through its installation and how to use it in different scenarios.
 
+## Table of Contents
```

Comment:
Hi @KanishkT123 We don't typically add a table of contents to the beginning of the VS Code topics but instead rely on the auto-generated right navigation, which displays the H2 headers.

![image](https://user-images.githubusercontent.com/12818376/153643256-595fcb44-5dc5-4730-a461-7b68bd427f38.png)

If you think a section needs to be elevated to an H2 for discoverability, we can change the header level. For example, your new section "Building Multi-Architecture builds" may make sense as an H2. I also think "Adding automation" and "devcontainer CLI build options" are discrete enough sections to be at their own H2 level.